### ISSUE TITLE
Hide BookCard tags when JS is enabled

### ISSUE THREAD

**Issue by rhernand3z** [NYPL/staff-picks#53]
Initially the book cards should contain the tags associated with each book. Once JS is enabled, the tags should be removed.

**Comment by rhernand3z** [comment]
Fixed by #96
